No longer runs - command -s not found

Using version 3.0.0:

Running the following command:
```cf-content-types-generator \
    -s $CONTENTFUL_SPACE_ID \
    -e $CONTENTFUL_ENVIRONMENT \
    -t $CONTENTFUL_MANAGEMENT_TOKEN \
    -o src/types/contentful
```

Generates this error:
```
(node:35016) [MODULE_NOT_FOUND] Warning: ModuleLoadError
module: @oclif/core@4.10.3
task: findCommand (Symbol(SINGLE_COMMAND_CLI))
plugin: cf-content-types-generator
root: /opt/homebrew/lib/node_modules/cf-content-types-generator
code: MODULE_NOT_FOUND
message: [MODULE_NOT_FOUND] require failed to load /opt/homebrew/lib/node_modules/cf-content-types-generator/lib/commands/Symbol(SINGLE_COMMAND_CLI): Cannot find module '/opt/homebrew/lib/node_modules/cf-content-types-generator/lib/commands/Symbol(SINGLE_COMMAND_CLI)'
Require stack:
- /opt/homebrew/lib/node_modules/cf-content-types-generator/node_modules/@oclif/core/lib/module-loader.js
- /opt/homebrew/lib/node_modules/cf-content-types-generator/node_modules/@oclif/core/lib/help/index.js
- /opt/homebrew/lib/node_modules/cf-content-types-generator/node_modules/@oclif/core/lib/errors/handle.js
- /opt/homebrew/lib/node_modules/cf-content-types-generator/node_modules/@oclif/core/lib/errors/index.js
- /opt/homebrew/lib/node_modules/cf-content-types-generator/node_modules/@oclif/core/lib/config/config.js
- /opt/homebrew/lib/node_modules/cf-content-types-generator/node_modules/@oclif/core/lib/config/index.js
- /opt/homebrew/lib/node_modules/cf-content-types-generator/node_modules/@oclif/core/lib/command.js
- /opt/homebrew/lib/node_modules/cf-content-types-generator/node_modules/@oclif/core/lib/index.js
- /opt/homebrew/lib/node_modules/cf-content-types-generator/bin/run
See more details with DEBUG=*
(Use `node --trace-warnings ...` to show where the warning was created)
/opt/homebrew/lib/node_modules/cf-content-types-generator/node_modules/@oclif/core/lib/config/config.js:415
            throw new errors_1.CLIError(`command ${id} not found`);
                  ^

CLIError: command -s not found
    at Config.runCommand (/opt/homebrew/lib/node_modules/cf-content-types-generator/node_modules/@oclif/core/lib/config/config.js:415:19)
    at async Object.run (/opt/homebrew/lib/node_modules/cf-content-types-generator/node_modules/@oclif/core/lib/main.js:97:16) {
  code: undefined,
  oclif: { exit: 2 },
  skipOclifErrorHandling: undefined,
  suggestions: undefined
}
```

Reverting back to 2.16.1 works correctly.

I get a similar error when running `cf-content-types-generator` without arguments.  The example above has it installed globally, but I get the same error regardless of where it is installed.

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -78,5 +78,8 @@ jobs:
       - name: Install Dependencies
         run: pnpm install --frozen-lockfile
 
+      - name: Build
+        run: pnpm build
+
       - name: Run Tests
         run: pnpm test:coverage

--- a/package.json
+++ b/package.json
@@ -23,8 +23,10 @@
   },
   "packageManager": "pnpm@9.15.9",
   "oclif": {
-    "default": ".",
-    "commands": "./lib/commands"
+    "commands": {
+      "strategy": "single",
+      "target": "./lib/commands/index"
+    }
   },
   "repository": {
     "type": "git",

